identity/app/views/profile/emailSettings.scala.html

```diff
@@ -53,10 +53,12 @@ <h2 class="form__heading">Email settings</h2>
                     </p>
                 </li>
                 <li>
-                    <button type="button" class="manage-account__button--mini manage-account__button--icon manage-account__button email-unsubscribe js-unsubscribe" data-link-name="identity : email : unsubscribe-all">
-                        Unsubscribe from all newsletters and marketing emails
-                        @fragments.inlineSvg("cross", "icon")
+                    <button type="button" class="manage-account__button--medium manage-account__button email-unsubscribe js-unsubscribe" data-link-name="identity : email : unsubscribe-all">
+                        Unsubscribe from all emails
                     </button>
+                    <div class="manage-account-unsubscribe__success js-unsubscribe-confirmation form__success">
```

Comment:
could you append this with js instead of having it hidden on page load?

Reply:
also make the alert <a href="https://github.com/guardian/frontend/pull/19779">polite</a>!!